install/centos-installation-guide.md

```diff
@@ -29,28 +29,10 @@ $ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo \
 $ sudo -E yum -y install kata-runtime kata-proxy kata-shim
 ```
 
-3. Configure Docker to use Kata Containers by default with the following commands:
+3. Install the upper layer (Docker or Kubernetes) and play with it
 
-```bash
-$ sudo mkdir -p /etc/systemd/system/docker.service.d/
-$ cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
-EOF
-```
-
-4. Restart the Docker systemd service with the following commands:
-
-```bash
-$ sudo systemctl daemon-reload
-$ sudo systemctl restart docker
-```
-
-5. Run Kata Containers
-
-You are now ready to run Kata Containers:
+If you want to use Kata Containers through Docker, click
+[here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#if-you-want-to-run-kata-containers-with-docker).
 
-```
-$ sudo docker run -ti busybox sh
-```
+If you want to use Kata Containers through Kubernetes, click
```

Comment:
Suggested rewrite: 

If you want to use Kata Containers through Kubernetes, see the "Developer Guide" [here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#if-you-want-to-run-kata-containers-with-kubernetes).